en/app.ftl

```diff
@@ -760,8 +760,11 @@ banner-choose-subdomain-description = You can make up any address @{ $subdomain
 success-subdomain-registered = Your domain @{ $subdomain } has been created
 # Variables:
 #   $subdomain (url) - User-set subdomain
+#   $username (url) - User-set username
```

Comment:
This is not how comments work in Fluent.

This comment (one `#`) is bound **only** to the next string. Two `##` makes it a group comment, which applies to all the other strings.

In this case, the comment should have been added to the string where this new variable appears.

Reply:
Thanks for fixing that up Francesco!